pyerrors/input/sfcf.py

```diff
@@ -29,266 +53,210 @@ def read_sfcf(path, prefix, name, **kwargs):
         b2b = 1
         single = 1
     else:
-        b2b = 0
+        if kwargs.get('b2b'):
+            b2b = 1
+        else:
+            b2b = 0
         single = 0
-
-    if kwargs.get('b2b'):
-        b2b = 1
-
+    if "replica" in kwargs:
+        reps = kwargs.get("replica")
+    if "files" in kwargs:
+        files = kwargs.get("files")
+
+    #due to higher usage in current projects, compact file format is default
+    compact = True
+    #get version string
+    version = "1.0"
```

Comment:
Also here I would make `version` a parameter with standard value.

Reply:
`version` is now also a parameter here, with default value `1.0`.